docs/monthly-meeting/2023-12.md

```diff
@@ -0,0 +1,79 @@
+- **Calendar event:** (send your e-mail to Mariatta for an invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [HackMD document](https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+## Roll call
+
+(Name / `@GitHubUsername` *[/ Discord, if different]*)
+- Petr Viktorin / `@encukou`
+- Daniele Procida / `@danieleprocida`
+- Hugo van Kemenade / `@hugovk`
+- Jim DeLaHunt / `@JDLH`
+- Ryan Duve / `@ryan-duve`
+- Ned Batchelder / `@nedbat`
+- Bradley / `@shenanigansd`
+- Ezio Melotti / `@ezio-melotti`
+- CAM Gerlach / `@CAM-Gerlach`
+- Usman Akinyemi / `@Unique-Usman`
+
+## Discussion
+
+* [Ryan] standard library documentation reference and adding types to it in a structured way.  For example, `collections.Counter.most_common` could be written
+
+      most_common(n: int) -> list[tuple[Any, int]]
+
+  [Petr] ... or `list[tuple[KeyType, int]]`?
+
+  - this has been discussed. There was opposition to adding the types to the stdlib code itself.
+  - [JDLH] I sometimes wish for more normative, less chatty module documentation. e.g. [*heapq* module](https://docs.python.org/3/library/heapq.html). e.g. old issue 29428 [*Doctest documentation unclear about multi-line fixtures*](https://bugs.python.org/issue29428).
+
+* [Petr] Use emoji to illustrate good and bad example commit messages
+  * https://devguide.python.org/getting-started/git-boot-camp/#accepting-and-merging-a-pull-request
+  * [python/devguide#1235](https://github.com/python/devguide/pull/1235)
+  * Should we do this for PEPs? see [python/docs-community#22](https://github.com/python/docs-community/issues/22)
+  * [Hugo] See also [python/peps#3567](https://github.com/python/peps/pull/3567) for green/red sidebars for good/bad example code in PEPs
+  * [JDLH] don't forget accessibility constraints when coming up with the style guide. e.g. some readers are red/green colorblind. ✅❌ are good in that they are legible even without color.
+
+* [Ryan] Thoughts about completing TOML->JSON conversion table: https://docs.python.org/3/library/tomllib.html#conversion-table and the TOML spec: https://toml.io/en/v1.0.0#spec
+
+
+## Reports and celebrations
+
+- PEP 732 ("The Python Documentation Editorial Board") has been submitted to the steering council [python/steering-council#220](https://github.com/python/steering-council/issues/220)
+
+- [Jim] FYI, Unicode Standard is changing form of authoritative standard documents from PDF to HTML, with corresponding change to document production tooling. If this is interesting I can provide more info. I am in the working group which is working on the new tooling.
```

Comment:
Shouldn't that use `[JDLH]`, like above? Or (my preference) use `[Jim]` throughout?

Reply:
We normally use names rather than usernames.

Reply:
@jdlh Which do you prefer?

Reply:
We normally use names rather than usernames. That is humane of us, and I am glad to join in. So therefore I am "Jim". In addition to being a username, "JDLH" is also what I use as initials (abbreviated signature). I appreciate you asking, Hugo @hugovk .